spec/ApiMailAdapter.spec.js

```diff
@@ -393,7 +393,7 @@ describe('ApiMailAdapter', () => {
       expect(payload.html).toBe(examplePayload.html);
     });
 
-    it('converts payload for AWS SES v3 SDK', () => {
+    it('converts payload for AWS SES (SDK v3)', () => {
       const payload = converter.awsSES(examplePayload);
```

Comment:
```suggestion
      const payload = converter.awsSes(examplePayload);
```

We use strict camel case, so even acronyms are only capitalized with first letter